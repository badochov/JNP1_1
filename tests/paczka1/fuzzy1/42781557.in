?

vtRK5 S890 70,2
JEzKt S394 46,0
vtRK5 S890 66,7
vtRK5 S890 49,4
S218 S145 6,7

JEzKt S145 74,1
JEzKt S394 64,4
JEzKt S394 26,4
JEzKt S394 6,9
JEzKt S218 13,0
JwX S890 67,2
Cj4ePYZ6 S218 68,1
JEzKt S218 26,9

Cj4ePYZ6 S218 88,5
JwX S890 20,8
JEzKt S145 42,7
Cj4ePYZ6 S890 5,2
JEzKt S145 161,4
?
?
Cj4ePYZ6 S890 64,7
S218 S145 55,4
S218 S394 26,2
Cj4ePYZ6 S890 66,0
JEzKt S145 98,9
JEzKt S145 461,9

?JEzKt
S218 S145 29,5
S218 S394 42,2
dUXI S890 7,3
JwX S890 99,2
dUXI S890 28,5
vtRK5 S218 67,7
JwX S890 86,3
Cj4ePYZ6 S218 34,6