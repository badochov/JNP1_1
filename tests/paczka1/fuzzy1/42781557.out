Cj4ePYZ6 S 20,4
JEzKt S 170,6
JwX S 46,4
vtRK5 S 3,5
S145 118,7
S218 34,3
S394 38,0
S890 49,9
Cj4ePYZ6 S 20,4
JEzKt S 170,6
JwX S 46,4
vtRK5 S 3,5
S145 118,7
S218 34,3
S394 38,0
S890 49,9
JEzKt S 533,6
